lib/plugins/aws/package/compile/events/apiGateway/lib/method/index.test.js

```diff
@@ -157,6 +164,163 @@ describe('#compileMethods()', () => {
     });
   });
 
+  it('should support inline model configuration', () => {
```

Comment:
As I suggested in other comment it'll be great to seclude logic into `lib/plugins/aws/package/compile/events/apiGateway/lib/requestValidators.js`, then `lib/plugins/aws/package/compile/events/apiGateway/lib/requestValidators.test.js` test file should have few tests configured through `runServerless` util.

Currently we're coming away from tests as you can see in most parts of the framework. Usually they focus on implementation internals and do not test against fully build `serverless` instance, which make them very weak.

See how new tests are written in those examples (sorry for not having a proper document on that yet, I will try to prepare one shortly):

- https://github.com/serverless/serverless/blob/7d3636f9682c7c9929a9061f105ed232d139aa56/lib/plugins/aws/package/compile/functions/index.test.js#L2624-L2719
- https://github.com/serverless/serverless/blob/7d3636f9682c7c9929a9061f105ed232d139aa56/lib/plugins/aws/package/compile/events/httpApi/index.test.js

You can find documentation for `runServerless` util here: https://github.com/serverless/test/blob/master/docs/run-serverless.md